lib/internal/crypto/keys.js

```diff
@@ -184,13 +226,17 @@ const [
       super('private', handle);
     }
 
-    export(encoding) {
+    export(options) {
+      if (options && options.format === 'jwk') {
+        const properties = this[kAsymmetricKeyJWKProperties]();
+        return this[kHandle].exportJwk(properties);
+      }
       const {
         format,
         type,
         cipher,
         passphrase
-      } = parsePrivateKeyEncoding(encoding, this.asymmetricKeyType);
+      } = parsePrivateKeyEncoding(options, this.asymmetricKeyType);
```

Comment:
Thought:

Crypto APIs should generally be designed to be "safe by default". Not necessarily secure, but at least somewhat safe. For example, other formats that don't support encryption (e.g., PKCS#1) throw an error when a `passphrase` is supplied, because silently ignoring the option potentially means that the user expects to obtain an encrypted key, but in reality, we'd be returning the key material without any protection.

Personally, I think it would be best to throw here if a passphrase is supplied and isn't supported for `'jwk'`. This is only relevant for `'private'` keys.

However, ignoring options by default is a fundamental flaw/feature of JavaScript and I won't tell you to change the behavior here. Similar problems exist in most other APIs, especially when considering differences across Node.js versions.

Reply:
✅